vtex-apps/address-locator
# Add component to show current and available addresses

#### What is the purpose of this pull request?

Add component to show current and available addresses

#### What problem is this solving?

- Show current selected address
- List available addresses (or previous addresses)
- Allow the user to change his address by inserting a new or selecting from available addresses
- Improve the overall flow of the components

#### How should this be manually tested?

- Go to [staging workspace](https://staging--delivery.myvtex.com/)
- Insert an address in the address locator
- After inserting, a bar will appear on the top of the page, below the header, showing your selected address
- Click on the bar to change the current address, by inserting a new one or selecting from available ones

#### Types of changes
